docs/appendices/release-notes/5.7.0.rst

```diff
@@ -79,6 +79,11 @@ SQL Standard and PostgreSQL Compatibility
 - Added the :ref:`scalar-current_role` scalar function as an alias for
   :ref:`scalar-current_user`.
 
+- Added the ``^`` operator as syntax sugar for the :ref:`scalar-power`
+  scalar function. E.g.::
+    SELECT 5 ^ 2;
+
```

Comment:
```suggestion
- Added the ``^`` operator as syntax sugar for the :ref:`scalar-power`
  scalar function. E.g.::

    SELECT 5 ^ 2;

```

Reply:
Because of 
> /home/docs/checkouts/readthedocs.org/user_builds/crate/checkouts/15688/docs/appendices/release-notes/5.7.0.rst:84: ERROR: Unexpected indentation.